src/darwin/Framework/CHIPTests/CHIPSetupPayloadParserTests.m

```diff
@@ -140,7 +140,7 @@ - (void)testOnboardingPayloadParser_NFC_NoError
     XCTAssertEqual(payload.setUpPINCode.unsignedIntegerValue, 2048);
     XCTAssertEqual(payload.vendorID.unsignedIntegerValue, 12);
     XCTAssertEqual(payload.productID.unsignedIntegerValue, 1);
-    XCTAssertFalse(payload.requiresCustomFlow);
+    XCTAssertEqual(payload.commissioningFlow, kCommissioningFlowStandard);
```

Comment:
Again, why the change from custom to standard?

Reply:
I changed to the standard whenever it was `XCTAssertFalse` before.

Reply:
Ah, I missed the "False".  Sorry...